tests/js/spec/components/sidebar/index.spec.jsx

```diff
@@ -293,12 +293,10 @@ describe('Sidebar', function() {
       wrapper.setProps({organization: sentry10Org});
       wrapper.update();
       const labels = wrapper.find('SidebarItemLabel').map(node => node.text());
-      expect(labels).toHaveLength(5);
+      expect(labels).toHaveLength(7);
       expect(labels).not.toContain('Assigned to me');
       expect(labels).not.toContain('Bookmarked issues');
       expect(labels).not.toContain('Recently viewed');
-      expect(labels).not.toContain('Activity');
-      expect(labels).not.toContain('Stats');
```

Comment:
Should we also have sidebar specs for the sentry10 behavior?

Reply:
This test is for sentry10 behaviour - do you mean for the other links on this page?

Reply:
nevermind I'm dumb and misread the diff.